core/src/com/unciv/models/metadata/GameParameters.kt

```diff
@@ -40,11 +40,32 @@ class GameParameters { // Default values are the default new game
         parameters.noBarbarians = noBarbarians
         parameters.oneCityChallenge = oneCityChallenge
         parameters.nuclearWeaponsEnabled = nuclearWeaponsEnabled
+        parameters.religionEnabled = religionEnabled
         parameters.victoryTypes = ArrayList(victoryTypes)
         parameters.startingEra = startingEra
         parameters.isOnlineMultiplayer = isOnlineMultiplayer
         parameters.baseRuleset = baseRuleset
         parameters.mods = LinkedHashSet(mods)
         return parameters
     }
+
+    // For debugging and MapGenerator console output
+    override fun toString() = "($difficulty $gameSpeed $startingEra, " +
+            "${players.count { it.playerType == PlayerType.Human }} ${PlayerType.Human} " +
+            "${players.count { it.playerType == PlayerType.AI }} ${PlayerType.AI} " +
+            "$numberOfCityStates CS, " +
+            sequence<String> {
+                if (isOnlineMultiplayer) yield("Online Multiplayer")
+                if (noBarbarians) yield("No barbs")
+                if (oneCityChallenge) yield("OCC")
+                if (!nuclearWeaponsEnabled) yield("No nukes")
+                if (religionEnabled) yield("Religion")
+                if (godMode) yield("God mode")
+                if (VictoryType.Cultural !in victoryTypes) yield("No ${VictoryType.Cultural} Victory")
+                if (VictoryType.Diplomatic in victoryTypes) yield("${VictoryType.Diplomatic} Victory")
+                if (VictoryType.Domination !in victoryTypes) yield("No ${VictoryType.Domination} Victory")
+                if (VictoryType.Scientific !in victoryTypes) yield("No ${VictoryType.Scientific} Victory")
```

Comment:
Iterate over victory types

Reply:
Debug output only, so to be silent on the defaults I went the quick copy&paste way and inverted the default-off one.... OK.